packages/builder/src/pages/builder/app/[application]/updating.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import Spinner from "components/common/Spinner.svelte"
+  import { API } from "api"
+
+  setInterval(async () => {
+    const response = await API.get({ url: "/api/migrations/status" })
+    if (!response.migrated) {
+      return
+    }
+
+    const urlParams = new URLSearchParams(window.location.search)
+    const returnUrl = urlParams.get("returnUrl")
+
+    window.location = returnUrl
```

Comment:
We could use routify's `redirect` here to redirect the user and prevent them going back to this page, rather than adding another entry to the browsers history by just navigating away again. Might also be sensible to add a fallback URL in case a return URL doesn't exist, to just push them to the main builder URL for that app or something.